src/pages/workspace/withPolicyAndFullscreenLoading.js

```diff
@@ -1,7 +1,8 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import {withOnyx} from 'react-native-onyx';
-import _ from 'underscore';
+import isEmpty from 'underscore/modules/isEmpty';
+import omit from 'underscore/modules/omit';
```

Comment:
Was testing size importing, this way the size is reduced from 20kb to 3kb, I can rollback if preferred

Reply:
I would suggest to use `lodash` helpers, because Expensify migrates from `underscore`.